rocAL/source/pipeline/master_graph.cpp

```diff
@@ -351,6 +383,12 @@ void MasterGraph::release() {
 
     if (_graph != nullptr)
         _graph->release();
+
+    // Release the graph created for each loader module, in case of multiple loaders
+    for (auto& graph : _graphs) {
```

Comment:
please add the condition in the for statement

Reply:
Done